staging/src/k8s.io/apiserver/pkg/server/options/network_proxy.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+	"github.com/spf13/pflag"
+
+	"k8s.io/apiserver/pkg/server"
+	"k8s.io/apiserver/pkg/server/networkproxy"
+)
+
+// NetworkProxyOptions holds the api server network proxy options.
+// See https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190226-network-proxy.md
+type NetworkProxyOptions struct {
+	// ConfigFile is the file path with api-server network proxy configuration.
+	ConfigFile string
+}
+
+// NewNetworkProxyOptions creates a new instance of NetworkProxyOptions
+//
+//  Provides the list of XXX that holds sane values
```

Comment:
XXX ?

Reply:
Is this trying to say it returns one with useful defaults?

Reply:
Fixed.